scapy/contrib/tacacs.uts

```diff
@@ -125,12 +125,12 @@ TacacsAuthorizationReply in pkt and pkt.status == 0  and pkt.arg_cnt is None and
 
 = filled values build -- SSH connection sample secret = test
 
-pkt = Ether()/IP()/TCP(dport=49)/TacacsHeader(seq=2, type=2 , flags=0, session_id=1391595252, length=20)/TacacsAuthorizationReply(status=1, arg_cnt=2, server_msg_len=0, data_len=0, arg_len_list=[11, 1])/TacacsPacketArguments(data='priv-lvl=15')/TacacsPacketArguments(data='a')
+pkt = Ether(src="00:00:00:00:00")/IP()/TCP(dport=49)/TacacsHeader(seq=2, type=2 , flags=0, session_id=1391595252, length=20)/TacacsAuthorizationReply(status=1, arg_cnt=2, server_msg_len=0, data_len=0, arg_len_list=[11, 1])/TacacsPacketArguments(data='priv-lvl=15')/TacacsPacketArguments(data='a')
```

Comment:
Why is this needed ? If this is needed, it seems that something was broken along Python 3 porting efforts.
This comment applies to similar modifications.